### PR TITLE
Update Kubernetes example

### PR DESCRIPTION
**Pull request by patrick-stephens**
Refreshed the K8S example to make sure it still functions.
Added documentation for it.

### CODE REVIEW

File: docs/modules/ROOT/pages/tutorial-kubernetes.adoc [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+[NOTE]
+====
+Unfortunately cluster monitor is currently designed for an on-premise solution so will not cope with auto-discovery on Kubernetes yet.
+Without this the various Grafana dashboards will not function correctly so you have to do it manually for now.
+====
+
+Kubernetes support is on the roadmap but, in the meantime, to manually add the cluster details you will need the credentials for accessing the REST API and a pod name for one of the Couchbase Server pods in the cluster (it will bootstrap the rest).
+
+These can then be added via the `Add Cluster` link on the landing page.
+Make sure to disable the `Add Nodes to Prometheus` option as this is handled already.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/tutorial-kubernetes.adoc:356]: Hm, I think I may have removed this in the latest iteration of the form - sounds like something we'll want to have optional (even if enabled by default).

**Review by markspolakovs — APPROVED** [review]

File: docs/modules/ROOT/pages/tutorial-kubernetes.adoc (hunk 1 above)
Comment (patrick-stephens) [reply docs/modules/ROOT/pages/tutorial-kubernetes.adoc:356]: Yeah we do, it still seems to work for me but maybe it is adding an unscrapeable target?

Comment (markspolakovs) [reply docs/modules/ROOT/pages/tutorial-kubernetes.adoc:356]: It'll be adding it as a separate target (check localhost:8080/prometheus/targets, you'll see something like `couchbase-server-managed-1`).

Comment (markspolakovs) [reply docs/modules/ROOT/pages/tutorial-kubernetes.adoc:356]: Filed https://issues.couchbase.com/browse/CMOS-156

**Pull request merged**